src/osmf/js/osmf_flash.js

```diff
@@ -391,6 +391,7 @@ require("../../../html5-common/js/utils/constants.js");
      */
     this.getCurrentTime = function() {
       this.callToFlash("getCurrentTime");
+      return currentTime;
```

Comment:
Yes, as this.callToFlash("getCurrentTime"); will call the function in AS which dispatches the event to JS and then return happens from the this.getCurrentTime